### DISCUSSION TITLE
Newbie: how to create assets consisting out of multiple components

### DISCUSSION THREAD

**Discussion by speedAmaster** [snipe/snipe-it#14074] [General]
I would like to manage assets which are consisting out of multiple components (like desktop-PC containing a CPU, motherboard, HDD, SSD, ...). Each is holding SerIDs, manufacturer, manual, buying date, warrenty, ...
How do I best manage this in snipe-it?

**Comment by mauro-miatello** [comment]
nice questions, I'm thinking about the same issue but I don't need a very detailed and updated summary, so I think I will create a custom field textarea "HW details" where I'llput a summary of all info I need created with some tool (e.g. Speccy or similar), or simply add the txt generated by the tool as attachment.

**Reply by speedAmaster** [reply]
I am thinking also to have a custom field to manually have a "textual" link to another assetID  ("assit is part of").
Still would be a significant extension to snipe-it.... IMHO

so I can add individual customer fields and attachments per asset item
